### PR TITLE
coreutils: bump to 8.30

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/sysutils/coreutils/patches/04-set-permission.c.patch [hunk 1]
```diff
@@ -1,38 +1,34 @@
-GNU Coreutils bug 21062 coreutils-8.24 - cp(1) check failures on tmpfs filesystem (Solaris 10 / Solaris 11)
```
Comment (pyhalov) [inline components/sysutils/coreutils/patches/04-set-permission.c.patch]: Please, preserve comments

**Comment by alarcher** [comment]
Done.

**Comment by pyhalov** [comment]
Test results differ here

+FAIL: tests/misc/timeout-parameters
+FAIL tests/misc/timeout-parameters.sh (exit status: 1)

**Comment by pyhalov** [comment]
Failed test case:
timeout 1.189731495357231765e+4932 true && echo 1 - should always echo 1
Also fails in current coreutils version.

**Comment by pyhalov** [comment]
Failed test is caused by https://www.illumos.org/issues/9858

**Pull request merged**